### DISCUSSION TITLE
Story Ideas

### DISCUSSION THREAD

**Discussion by WizardMaster38** [WizardMaster38/Infernum-Ad-Infinitum-To-Hell-and-Beyond#9] [Ideas]
Story Ideas go in **seperate comments**.

**Comment by deltasolidsnake** [comment]
Idea 1. A portal to Hell opens and releases stereotypical demons into the world, and it is up to the characters to fight back the evil creatures from whence they came. This would be the more cheery and fun story.

**Reply by WizardMaster38** [reply]
*DOOM plot 101*

**Comment by deltasolidsnake** [comment]
Idea 2. Following a mistake at a great important ritual, a portal to Hell is opened, demons, infections, zombies, and other paranormal cryptids are invading the world. The story would involve the characters fighting back the invasion, encountering the abominations that came through from hell. They could be plagued by hallucinations and voices of Hell's ruler, think of the Gravemind from Halo 3. This option is the horror option and would allow for more creative freedom and more interesting stories.

**Reply by WizardMaster38** [reply]
I like this idea, but remember that this is all going to be *text based*.

**Reply by deltasolidsnake** [reply]
Yes, I know. I'm good at writing horror.

**Reply by deltasolidsnake** [reply]
I'm good at being descriptive.

**Reply by WizardMaster38** [reply]
True that
